README.md

```diff
@@ -192,7 +192,7 @@ Directories:
   │       │
   │       │   ┌───────────────┐  - Broadcast of data to all parties in the network
   │       ├───┤ broadcast [Bm]│    * Implements dispatcher for batch component
-  │       │   │ manager       |    * Integrates with public storage interface [Ps] plugin
+  │       │   │ manager       |    * Integrates with shared storage interface [Ps] plugin
```

Comment:
Do we change the "Ps" abbreviation as well?

Reply:
Yeah - I need to go through and give this whole thing a bit of love at some point soon 

```
       ,
      /(  ___________
     |  >:===========`
      )(
      ""
```

Reply:
It took me entirely too long to read what that meant 😆